### PR TITLE
internal: fix typos

### PR DESCRIPTION
**Pull request by Iceber**
Signed-off-by: IceberGu wei.cai-nat@daocloud.io

### CODE REVIEW

File: dialoptions.go [hunk 1]
```diff
@@ -473,7 +473,7 @@ func WithStreamInterceptor(f StreamClientInterceptor) DialOption {
 }
 
 // WithChainStreamInterceptor returns a DialOption that specifies the chained
-// interceptor for unary RPCs. The first interceptor will be the outer most,
+// interceptor for stream RPCs. The first interceptor will be the outer most,
```
Comment (easwars) [inline dialoptions.go:476]: s/stream/streaming

**Comment by easwars** [comment]
@Iceber The CLA bot is not reporting any status. Could you please sign the CLA if you haven't already. Thanks.

**Comment by Iceber** [comment]
I signed it

**Comment by Iceber** [comment]
I signed it

**Comment by easwars** [comment]
Closing to kick the CLA-bot.

**Comment by easwars** [comment]
I really don't understand what the problem is with the CLA bot :(

**Comment by WillsonHG** [comment]
Hello @easwars @Iceber  Please try closing and reopening this same pull request again. We have just deployed a fix for this and the status should report now. Thank you!

**Comment by easwars** [comment]
> Hello @easwars @Iceber Please try closing and reopening this same pull request again. We have just deployed a fix for this and the status should report now. Thank you!

Doesn't look like closing and reopening has helped in this case :(

**Comment by WillsonHG** [comment]
@easwars  We've just pushed another fix for this, please try to close and reopen again. Sorry for all the trouble!

**Comment by easwars** [comment]
Still seems to be stuck on `Waiting for status to be reported`.

**Comment by WillsonHG** [comment]
Hello @easwars Will you please try to close and reopen again? Thanks!

**Comment by easwars** [comment]
@Iceber : It looks like you have not signed the CLA. Could you please sign it.

**Comment by easwars** [comment]
@WillsonHG : Looks like it is working now. Thanks.

**Pull request merged**